ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -51,12 +51,50 @@ const checkFilePreviewInFileAttachment = (component) => {
 const deleteSelectedFiles = (component, fileNames) => {
     cy.get(component).then(() => {
         fileNames.forEach((fileName) => {
-            cy.get(".cmp-adaptiveform-fileinput__filedelete").should('have.attr', 'role', 'button');
```

Comment:
While you have removed this, you are no longer testing v1, v2 and v3. In the changelist, ideally fileinput.runtime.spec.js spec should remain as it is, and you need to add test spec for other versions

Reply:
For now can we add a check to see if the element is not button then it should have a role button attribute.

To run separate test cases against v1 v2 and v3, we will need to create separate test collaterals for v1 v2 and v3 such that each test case will load its version of the form. Maybe we can create a separate PR for this

Reply:
Implemented!

Reply:
@rismehta initially delete button was span and had attribute role="button", now in V3 delete button has been created using <button> tag and removed role="button" and this is done using JS, this is the reason for changing fileinput.runtime.js file because V3 doesn't has span in delete button

Reply:
for v3, there has to be a separate test suite, existing test suite should not change. Please incorporate the same and create a new PR